src/components/List/List.tsx

```diff
@@ -64,6 +71,7 @@ class List extends UIComponent<Extendable<ListProps>, ListState> {
     accessibility: PropTypes.func,
     debug: PropTypes.bool,
     items: customPropTypes.collectionShorthand,
+    innerRef: PropTypes.func,
```

Comment:
Didn't we decide that we will use the Ref component instead of adding this? Are you updated to latest master?

Reply:
We need that prop in the component already, so downshift callback prop will get the actual html node and add its stuff to it, here: `{...getMenuProps({ refKey: 'innerRef' })}` . If I don't specify it in the component, it won't work. I am doing the <Ref> magic in Dropdown as an HOC, but I still need this to be available as a handled prop. @kuzhelov @mnajdova

Reply:
let me help you with that - we definitely shouldn't expose this prop for now for this component only, as we have workarounds at the moment. Refs for components, if they would be exposed to the API, should be exposed for all in as single step, for the sake of preserving API consistency